### ISSUE TITLE
Can't get right results for simple usage example

### ISSUE THREAD

**Issue by Jiluzhang** [aryeelab/hichipper#16]
I tried to check if hichipper installed sucessfully by running simple usage example. My command is "hichipper --out output1 yaml/one.yaml", but it looks like existing some problems. The output is as following.
  Tue May 23 12:40:26 CST 2017: Starting hichipper pipeline v0.6.0
Tue May 23 12:40:26 CST 2017: Parsing user parameters
Anchors removed due to excessive size (likely at ends of chromosomes): 0
Tue May 23 12:40:29 CST 2017: Processing dSRR3467177
Tue May 23 12:40:29 CST 2017: Total_PETs=8658944
Tue May 23 12:40:29 CST 2017: Mapped_unique_quality_pairs=156192
Tue May 23 12:40:29 CST 2017: Mapped_unique_quality_valid_pairs=56517
Tue May 23 12:40:29 CST 2017: Intersecting PETs with anchors
Tue May 23 12:40:29 CST 2017: Finished the anchor merging.
Tue May 23 12:40:33 CST 2017: Intrachromosomal_valid_small=8811
Tue May 23 12:40:33 CST 2017: Intrachromosomal_valid_med=21314
Tue May 23 12:40:33 CST 2017: Intrachromosomal_valid_large=3640
Tue May 23 12:40:33 CST 2017: Total number of anchors used: 926
Tue May 23 12:40:33 CST 2017: Total number of reads in anchors: 27298
Tue May 23 12:40:33 CST 2017: Mapped_unique_intra_quality_anchor=1138
Tue May 23 12:40:33 CST 2017: Mapped_unique_intra_quality_anchor_small=763
Tue May 23 12:40:33 CST 2017: Mapped_unique_intra_quality_anchor_med=352
Tue May 23 12:40:33 CST 2017: Mapped_unique_intra_quality_anchor_large=23
Tue May 23 12:40:33 CST 2017: Loop_PETs=352
Tue May 23 12:40:33 CST 2017:
['Rscript', u'output1/qcReport.R', '/home/students/pinlv/Luz/python/lib/python2.7/site-packages/hichipper', 'output1', '/home/students/pinlv/Luz/hichipper/hichipper/tests', '0.6.0', 'dSRR3467177']
/home/students/pinlv/Luz/hichipper/hichipper/tests

processing file: qcReport_make.Rmd
  |...                                                              |   5%
   inline R code fragments

  |.......                                                          |  11%
label: unnamed-chunk-1 (with options)
List of 3
 $ echo   : logi FALSE
 $ message: logi TRUE
 $ warning: logi TRUE

  |..........                                                       |  16%
   inline R code fragments

  |..............                                                   |  21%
label: unnamed-chunk-2 (with options)
List of 3
 $ echo   : logi FALSE
 $ message: logi FALSE
 $ warning: logi FALSE

  |.................                                                |  26%
  ordinary text without R code

  |.....................                                            |  32%
label: unnamed-chunk-3 (with options)
List of 4
 $ echo   : logi FALSE
 $ message: logi FALSE
 $ warning: logi FALSE
 $ results: chr "asis"

  |........................                                         |  37%
  ordinary text without R code

  |...........................                                      |  42%
label: unnamed-chunk-4 (with options)
List of 4
 $ echo     : logi FALSE
 $ message  : logi FALSE
 $ warning  : logi FALSE
 $ out.width: chr "\\textwidth"

  |...............................                                  |  47%
  ordinary text without R code

  |..................................                               |  53%
label: unnamed-chunk-5 (with options)
List of 4
 $ echo     : logi FALSE
 $ message  : logi FALSE
 $ warning  : logi FALSE
 $ out.width: chr "\\textwidth"

  |......................................                           |  58%
  ordinary text without R code

  |.........................................                        |  63%
label: unnamed-chunk-6 (with options)
List of 6
 $ echo     : logi FALSE
 $ message  : logi FALSE
 $ warning  : logi FALSE
 $ results  : chr "asis"
 $ out.width: chr "\\textwidth"
 $ fig.width: num 7

  |............................................                     |  68%
  ordinary text without R code

  |................................................                 |  74%
label: unnamed-chunk-7 (with options)
List of 4
 $ echo     : logi FALSE
 $ message  : logi FALSE
 $ warning  : logi FALSE
 $ out.width: chr "\\textwidth"

Using count as value column: use value.var to override.
  |...................................................              |  79%
  ordinary text without R code

  |.......................................................          |  84%
label: unnamed-chunk-8 (with options)
List of 4
 $ echo     : logi FALSE
 $ message  : logi FALSE
 $ warning  : logi FALSE
 $ out.width: chr "\\textwidth"

  |..........................................................       |  89%
  ordinary text without R code

  |..............................................................   |  95%
label: unnamed-chunk-9 (with options)
List of 4
 $ echo     : logi FALSE
 $ message  : logi FALSE
 $ warning  : logi FALSE
 $ out.width: chr "\\textwidth"

  |.................................................................| 100%
  ordinary text without R code

output file: qcReport_make.knit.md

/home/students/pinlv/bin/pandoc +RTS -K512m -RTS qcReport_make.utf8.md --to html --from markdown+autolink_bare_uris+ascii_identifiers+tex_math_single_backslash --output output1.hichipper.qcreport.html --smart --email-obfuscation none --self-contained --standalone --section-divs --template /home/students/pinlv/R/x86_64-pc-linux-gnu-library/3.3/rmarkdown/rmd/h/default.html --highlight-style pygments --css /home/students/pinlv/R/x86_64-pc-linux-gnu-library/3.3/rmarkdown/rmarkdown/templates/html_vignette/resources/vignette.css --include-in-header /tmp/RtmpDfS9nh/rmarkdown-str85202723e301.html --mathjax --variable 'mathjax-url:https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML'

Output created: output1.hichipper.qcreport.html
Warning messages:
1: Transformation introduced infinite values in continuous x-axis
2: Removed 751 rows containing non-finite values (stat_bin).
Processing: dSRR3467177
Error in { : task 1 failed - "missing value where TRUE/FALSE needed"
Calls: %do% -> <Anonymous>
In addition: There were 35 warnings (use warnings() to see them)
Execution halted

  I think there are problems with calling interaction, but I don't know how to solve them. Could you help me to deal with these bugs? Thanks so much!
Luzhang Ji

**Comment by caleblareau** [comment]
Hi Luzhang--

The issue that you reported is from the mango FDR computation failing as there are too few loops in the sample data to fit the full parametric model. This was something that I’m aware of, and I think that you should be getting still reasonable output (.bedpe files) that show that the tool is working. However, I just need to make the sample data deeper to accommodate the new anchor calling (which I've been resisting doing simply to keep the file sizes small). 

I will try to get some cleaner test data up this weekend and will update this thread when its complete. Thanks for the report!

**Comment by Jiluzhang** [comment]
Hi caleblareau,
When I imported my own data to hichipper, the output was still the same. So what should I do to compare different samples? I think diffloop is not suitable.

**Comment by caleblareau** [comment]
@Jiluzhang sorry for the late response on this, as of commit `17c8106` (with fixed build at `deed89c`), I've changed the simple usage example to a larger file set that allows all functionality to work (but it does take a longer time). This should include a .rds, .html, and .mango files that were previously not being generated as the loop calls were too sparse. 

If you aren't getting the correct output at this point, then I'd venture to guess you are missing a dependency.
